lesson_06/homeworks_06.py

```diff
@@ -36,6 +72,12 @@
 Сума цифр числа 12345: 15
 """
 
+number_input = input('Enter your number: ')
+number_to_calculate = list(str(int(number_input)))
```

Comment:
```suggestion
number_input = input('Enter your number: ')
number_to_calculate = list(number_input)
```
input повертає завжди строку